openbb_terminal/core/models/sources_model.py

```diff
@@ -1,18 +1,43 @@
+import json
 from typing import Dict
 
-from pydantic import Field
-from pydantic.dataclasses import dataclass
+from pydantic.dataclasses import Field, dataclass
 
+from openbb_terminal.core.config.paths import DATA_SOURCES_DEFAULT_FILE
 from openbb_terminal.core.models.base_model import BaseModel
+from openbb_terminal.core.sources.utils import flatten
 
 # pylint: disable=useless-parent-delegation
 
 
+def read_default_sources() -> Dict:
+    """Read default sources from file.
+
+    Returns
+    -------
+    Dict
+        Dictionary with sources
+    """
+    try:
+        with open(DATA_SOURCES_DEFAULT_FILE) as file:
+            return flatten(json.load(file))
+    except Exception as e:
+        print(
```

Comment:
```suggestion
        console.print(
```

Reply:
This yields circular import, because the module `rich_config.py` where `console` lives is accessing the `current_user.py`. Anyways that's a good idea and we should fix this, but it falls a bit outside of the PR scope

Reply:
Got it. Perfect - we'll fix it later, if anything.